src/tables.rs

```diff
@@ -4582,3 +4582,1461 @@ pub const TITLE: &'static [(u32, &'static [u32])] = &[
     (125250, &[125216]),
     (125251, &[125217]),
 ];
+
+#[allow(dead_code)]
+pub const CASE_FOLDING_SIMPLE: &'static [(u32, u32)] = &[
```

Comment:
I assume this came from something like `yeslogic-ucd-generate case-folding-simple /home/wmoore/Downloads/ucd-14.0`, It would probably best if this data was in it's own file so the header about how it was generated is included.